WordPress/src/main/java/org/wordpress/android/ui/sitecreation/previews/SitePreviewViewModel.kt

```diff
@@ -172,7 +172,7 @@ class SitePreviewViewModel @Inject constructor(
     private fun getCleanUrl(url: String) = StringUtils.removeTrailingSlash(urlUtils.removeScheme(url))
 
     private fun createSitePreviewData(): UrlData {
-        val url = domainName
+        val url = result?.let { getCleanUrl(it.site.url) ?: "" } ?: ""
```

Comment:
Elvis enveloping another Elvis!  Would it be better to separate them to something like function param?

Reply:
This is one of those situations where I'm weighing my options. Yes, it looks dumb 😅 . But it's such a short 1 liner. It doesn't even seem worth the effort to expand into multiple lines to not see `?: ""` an extra time. If we were doing that in multiple places I'd of course extract into a function. That being said, if you feel strongly against it I will make the change @ravishanker . I leave the final decision up to you.